javascript/builtins/Intl.json

```diff
@@ -830,7 +830,7 @@
                     "version_added": "35"
                   },
                   "chrome_android": {
-                    "version_added": null
+                    "version_added": "<66"
```

Comment:
The "less than" syntax isn't valid as far as I know, also I'm not sure what the policy is on determining version support this way. I assume we'd want precision? This should probably just be set to `true` for now if we don't know the version it was added in.

Reply:
Okay, that would work then. (I realized after posting that "<=" is actually what this ought to mean; it works in 66 and may work in earlier versions.) Having proven that it DOES work, I want the next person to be able to see this looking encouraging, not scary.

Reply:
Hi, thanks for contributing to mdn-browser-compat-data! :tada:  @connorshea is right, in this case we would put `true`, because we now it is supported but we don't know in which version support started. I've changed this real quick.